source/tutorial/triggers-fts.txt

```diff
@@ -0,0 +1,365 @@
+      alert box when clicked:
+
+      - At the top of the ``ItemListView`` function block, add a ``useState()`` hook 
+        to keep track of the alert box view:
+
+        .. literalinclude:: /tutorial/backend-examples/ItemListView.snippet.showAlertBox.tsx
+           :language: javascript
+           :caption: src/ItemListView.tsx
+           :emphasize-lines: 7
+
+      - After the ``Add To-Do`` button in the main view, add an ``Alerts`` button that 
+        toggles the alert box overlay:
+
+        .. literalinclude:: /tutorial/backend-examples/ItemListView.snippet.alertOverlayButton.tsx
+           :language: javascript
+           :caption: src/ItemListView.tsx
+           :emphasize-lines: 19-41
+
+      - To change the size and color of the button, add the following lines 
+        to the ``styles`` block at the bottom of the file:
+
+        .. literalinclude:: /tutorial/backend-examples/ItemListView.snippet.alertButtonStyle.tsx
+           :language: javascript
+           :caption: src/ItemListView.tsx
+           :emphasize-lines: 7-11
+
+Store Alert Terms in Atlas
+--------------------------
+
+Now that you've created the frontend component for the alert box, 
+configure the application's backend to store and keep track of your 
+alert terms in Atlas.
+
+.. procedure::
+
+   .. step:: Create a New Collection
+
+      Create a collection to store the terms that users enter in the app.
+      In the Atlas UI: 
+
+      A. Click the :guilabel:`Data Services` tab.
+      
+      #. For the deployment that's synced to the template app, 
+         click :guilabel:`Browse Collections`.
+      
+      #. In the left navigation, click the :guilabel:`+` icon next to the ``todo`` 
+         database to add a new collection.
+      
+      #. Name the collection ``alerts``, then click :guilabel:`Create` to save 
+         the collection.
+
+   .. step:: Enable Access to the Collection
+
+      After creating the collection, you must give your app the necessary permissions
+      to write to the ``todo.alerts`` collection:
+
+      A. Click the :guilabel:`App Services` tab.
+      
+      #. Click the tile for your app.
+
+      #. In the left navigation under :guilabel:`Data Access`, click :guilabel:`Rules`.
+
+      #. Under the ``todo`` database, click the ``alerts`` collection.
+      
+      #. In the right dialog box, select the :guilabel:`readAndWriteAll` preset role.
+      
+      #. Click :guilabel:`Add preset role` to confirm your selection.
+
+      #. By default, your application enables deployment drafts. To 
+         :ref:`manually deploy <deploy-ui>` your changes, click 
+         :guilabel:`Review Draft & Deploy` and then :guilabel:`Deploy`.
+         
+   .. step:: Write to the Collection
+
+      Once you've configured write access to the ``todo.alerts`` collection, return
+      to your application code.
+
+      In ``src/ItemListView.tsx``, add the following lines to the top of tbe function block
```

Comment:
small nitpick - typo here t**b**e instead of the

Reply:
Good catch - thanks!!